src/binary.ts

```diff
@@ -337,7 +333,7 @@ export class UUID extends Binary {
       );
     }
     super(bytes, BSON_BINARY_SUBTYPE_UUID_NEW);
-    this[kId] = bytes;
+    this.buffer = bytes;
```

Comment:
We can remove this altogether now since the buffer will be assigned in super, right?

Reply:
I'm not sure since UUID.generate also uses the bytes variables